examples/getting-started/Vagrantfile

```diff
@@ -3,9 +3,9 @@
 
 Vagrant.require_version ">= 2.0.0"
 
-cilium_version = (ENV['CILIUM_VERSION'] || "v1.7.2")
+cilium_version = (ENV['CILIUM_VERSION'] || "v1.9")
```

Comment:
The environment variables `CILIUM_VERSION` and `CILIUM_TAG` are never defined nor explained in the guide. It probably makes sense to explain the and maybe update `vagrant up` instructions for `CILIUM_VERSION=1.9 vagrant up` or something.

Also, I think that occurrences of `CILIUM_TAG` could actually be replaced by `CILIUM_VERSION` and `CILIUM_TAG` subsequently be removed altogether.

Reply:
Updated the `Vagrantfile` and the docs and tested with:
```
$ CILIUM_VERSION=v1.8 vagrant up
..
$ vagrant ssh
..
vagrant@vagrant:~/cilium/examples/getting-started$ cilium version
Client: 1.8.4 c2442da08 2020-10-20T11:13:53+02:00 go version go1.14.10 linux/amd64
Daemon: 1.8.4 c2442da08 2020-10-20T11:13:53+02:00 go version go1.14.10 linux/amd64
```